### PR TITLE
[docs] update forking instructions

### PR DESCRIPTION
**Pull request by tsmethurst**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -101,15 +101,15 @@ See [this blog post](https://blog.sgmansfield.com/2016/06/working-with-forks-in-
...
-> Fork the repository on GitHub or set up whatever other remote git repo you will be using. In this case, I would go to GitHub and fork the repository.
+> `mkdir -p ~/go/src/github.com/superseriousbusiness && git clone git@github.com:superseriousbusiness/gotosocial ~/go/src/github.com/superseriousbusiness/gotosocial`
 >
-> Navigate to the top level of the repository on your computer. Note that this might not be the specific package you’re using:
+> Navigate to the top level of the upstream repository on your computer:
 >
-> `cd $GOPATH/src/github.com/superseriousbusiness/gotosocial`
+> `cd ~/go/src/github.com/superseriousbusiness/gotosocial`
 >
 > Rename the current origin remote to upstream:
```
Comment (igalic) [inline CONTRIBUTING.md:114]: wouldn't it be easier to clone your own fork first, or am i missing something?

**Review by igalic — COMMENTED** [review]
❓

File: CONTRIBUTING.md (hunk 1 above)
Comment (tsmethurst) [reply CONTRIBUTING.md:114]: I guess you could create the superseriousbusiness dir and then clone your own fork into it instead, it doesn't really make that much difference i think

Comment (igalic) [reply CONTRIBUTING.md:114]: ah, because `$GOPATH`…

**Pull request merged**